src/components/forms/OpaqueIngressSecretForm.js

```diff
@@ -138,24 +169,30 @@ class OpaqueIngressSecretForm extends Component {
             value={this.state.secrets_group}
             onChange={this.handleInputChange}
             invalidCallback={() =>
-              this.props.secretsManagerGroupCallback(this.state, this.props)
+              this.props.secretsManagerGroupCallback("secrets_group")(
```

Comment:
we should not be hardcoding `"secrets_group` here, this callback should only take two arguments, `stateData` and `componentProps`

Reply:
we can have a third prop for name that can be hard coded